### PR TITLE
Kathleen

### PR DESCRIPTION
**Pull request by kathleen-carroll**

### CODE REVIEW

**Comment by kathleen-carroll** [comment]
@ingredients = @chef.dishes.joins(:ingredients).distinct.pluck('ingredients.name')

this is what i should've put for user story 3.. I forgot to use active record but just figured it out

File: app/models/dish.rb [hunk 1]
```diff
@@ -1,4 +1,16 @@
 class Dish <ApplicationRecord
   validates_presence_of :name, :description
   belongs_to :chef
+  has_many :dish_ingredients
+  has_many :ingredients, through: :dish_ingredients
+
+  def ingredient_names
```
Comment (mikedao) [inline app/models/dish.rb]: we dont really need this, we can just iterate through the collection in the view.

File: app/models/dish.rb [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+    ingredients.map do |ingredient|
+      ingredient.name
+    end
+  end
+
+  def total_calories
+    ingredients.sum {|ingredient| ingredient.calories}
```
Comment (mikedao) [inline app/models/dish.rb]: Use AR when you can

File: app/views/chefs/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<h1><%= @chef.name %></h1>
+
+<p>Most Popular Ingredients: <%= @chef.most_popular %></p>
+<%=link_to "See Ingredients", "/chefs/#{@chef.id}/ingredients", method: :get%>
```
Comment (mikedao) [inline app/views/chefs/show.html.erb]: the get is not necessary, links get by default

File: app/models/dish.rb (hunk 2 above)
Comment (mikedao) [reply app/models/dish.rb]: also, needs a model test

**Pull request closed without merging**